### PR TITLE
OSDOCS-10971: updates relnotes file 4.17 MicroShift

### PR DESCRIPTION
**Pull request by aravipra**
Version(s):
4.17

Issue:
[OSDOCS-10971](https://issues.redhat.com/browse/OSDOCS-10971)

Link to docs preview:
[relnotes file](https://78356--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-17-release-notes.html)

QE review:
- N/A docs plumbing

### CODE REVIEW

File: microshift_release_notes/microshift-4-17-release-notes.adoc [hunk 1]
```diff
@@ -34,85 +34,76 @@ This release adds improvements related to the following components and concepts.
 
 [id="microshift-4-17-updating_{context}"]
 === Updating
-Updating two minor versions in a single step is now supported. Updates for both single-version minor releases and patch releases are still supported.
+//Updating two minor versions in a single step is supported from 4.16. Updates for both single-version minor releases and patch releases are still supported.
```
Comment (mburke5678) [inline microshift_release_notes/microshift-4-17-release-notes.adoc:37]:
```suggestion
//Updating two minor versions in a single step is supported in {what} 4.16 and higher. Updates for both single-version minor releases and patch releases are still supported.
```

**Comment by mburke5678** [comment]
@aravipra I am not sure why I am reviewing this PR at this time.

File: microshift_release_notes/microshift-4-17-release-notes.adoc (hunk 1 above)
Comment (aravipra) [reply microshift_release_notes/microshift-4-17-release-notes.adoc:37]: line 37 is commented. So the contents of this line are not visible in the preview. No changes are required in this line as it is commented.

**Comment by aravipra** [comment]
> @aravipra I am not sure why I am reviewing this PR at this time.

@mburke5678 I am updating the 4.17 release notes. Content that are commented out are retained as I might need to make edits later.

**Pull request merged**